snooty/types.py

```diff
@@ -16,6 +16,7 @@
     Union,
 )
 
+import imagesize  # type: ignore
```

Comment:
It would be better to include a stub file in `stubs/`: e.g. https://github.com/mongodb/snooty-parser/blob/main/stubs/bson.pyi

Reply:
good catch! done